### PR TITLE
temporary workaround for ubuntu archive being down

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Review by lluiscampos — APPROVED** [review]

**Comment by oleorhagen** [comment]
Works, I'm merging

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,10 @@ ARG DISTRO=debian
 ARG VERSION=buster
 FROM $DISTRO:$VERSION
 
+# TODO - remove - Temporary workaround for Ubuntu archive being down
+RUN grep 'archive.ubuntu.com' /etc/apt/sources.list && sed -i.bak 's/archive.ubuntu.com/us.archive.ubuntu.com/' /etc/apt/sources.list; \
```
Comment (merlin-northern) [inline Dockerfile:6]: -F here would be nice

Comment (oleorhagen) [reply Dockerfile:6]: Yeh, mby. But it's literally only gonna be here for a few hours :)

**Comment by kacf** [comment]
@merlin-northern: Don't forget to revert this change.

**Pull request merged**